src/error.jl

```diff
@@ -1,41 +1,42 @@
-using SumTypes: @sum_type
-
 export get_error_code, get_error_message
 
-# See https://www.mcobject.com/docs/Content/Programming/C/Return_Codes.htm or
-# https://www.gnu.org/software/libc/manual/html_node/Exit-Status.html
-"Represent various return codes from the Spglib library."
-@sum_type SpglibReturnCode begin
-    SUCCESS
-    SPACEGROUP_SEARCH_FAILED
-    CELL_STANDARDIZATION_FAILED
-    SYMMETRY_OPERATION_SEARCH_FAILED
-    ATOMS_TOO_CLOSE
-    POINTGROUP_NOT_FOUND
-    NIGGLI_FAILED
-    DELAUNAY_FAILED
-    ARRAY_SIZE_SHORTAGE
-    NONE
-end
+# Mapping of the `SpglibError` C enum variants to integer values.
+const SPGLIB_SUCCESS = 0
+const SPGERR_SPACEGROUP_SEARCH_FAILED = 1
+const SPGERR_CELL_STANDARDIZATION_FAILED = 2
+const SPGERR_SYMMETRY_OPERATION_SEARCH_FAILED = 3
+const SPGERR_ATOMS_TOO_CLOSE = 4
+const SPGERR_POINTGROUP_NOT_FOUND = 5
+const SPGERR_NIGGLI_FAILED = 6
+const SPGERR_DELAUNAY_FAILED = 7
+const SPGERR_ARRAY_SIZE_SHORTAGE = 8
+const SPGERR_NONE = 9
```

Comment:
These error codes (1 .. 9) aren't currently used. Removing them from the PR might make sense, at your disgression.